.gitmodules

```diff
@@ -45,6 +45,7 @@
 [submodule "src/sonic-platform-common"]
 	path = src/sonic-platform-common
 	url = https://github.com/Azure/sonic-platform-common
+	branch = 201811
```

Comment:
I don't think it is necessary to put branch here. It is just that it will checkout a separate branch on top of the commit-id based de-attached HEAD.  But it is fine to keep it.

Reply:
Thanks Zhenggen, there has been some commits that I don't think I should pick up. :-)